packages/theming/src/utils/arrowStyles.ts

```diff
@@ -30,9 +30,13 @@ export type ARROW_OPTIONS = {
 
 const animationStyles = (position: ARROW_POSITION, modifier: string) => {
   const property = position.split('-')[0];
+  /**
+   * 1. Prevent bleed-through on revised stacking context (i.e. parent transform)
+   */
   const animationName = keyframes`
     0%, 66% {
       ${property}: 2px;
+      border: transparent; /* [1] */
```

Comment:
So this was the trick when I first moved over to `transform: translate` for the animation. Good find.